### DISCUSSION TITLE
[Bug] When trying to get an I.D, the message wont show and its just blank

### DISCUSSION THREAD

**Discussion by uhtheuhuglybeaner** [danthe1st/Mee6-bypasser#26] [Support]
**Describe the bug**

**To Reproduce**
e.g.

**Expected behavior**

**Screenshots**

**Server (if you run the Bot by yourself, please complete the following information about the Service running the Software):**
 - OS: [e.g. Ubuntu]
 - If possible, include arguments to the JVM and the Program(not the Bot token), logs(for example if you redirected stdout and stderr)

**Client (please complete the following information):**
 - OS: [e.g. Windows 10]
 - Did you use the Discord App or a Web-Browser [e.g. Google Chrome]

**Additional context**

**Comment by danthe1st** [comment]
What exactly did you do? Can you share a screenshot?
Please fill out the issue template and not only the title.
